src/main/java/newuser/NewUserController.java

```diff
@@ -0,0 +1,24 @@
+package newuser;
+
+import entities.FacilityType;
```

Comment:
to avoid confusion w CA, remove the imports for entities from here since you don't reference them anymore